src/IotCentral/IotCentral/ChangeLog.md

```diff
@@ -19,6 +19,10 @@
 -->
 ## Upcoming Release
 
+## Version 1.0.0
+* Stable release version, updated to IoT Central .NET Management Plane SDK version 4.0.0. 
+* This change introduces support for System-Assigned Managed Identities, adds support for regional locations like 'eastus' (while removing support for geographic locations like 'unitedstates'), and removes support for legacy S1 sku.
```

Comment:
NIT: Support for regional locations is existing behavior in api version 2018-09-01, the only difference is deprecating support for geographical resources.

Reply:
Updated changelog, thanks.